tensorflow-notebook/templates/NOTES.txt

```diff
@@ -0,0 +1,9 @@
+1. Get the application URL by running these commands:
```

Comment:
I'd make the notes specific to okteto cloud as well. maybe just mention that it was installed successfully and a hint to go to the UI to get the URL?

Reply:
Ok, I add it.